resources/i18n/it_IT/cura.po

```diff
@@ -4607,21 +4213,10 @@ msgid "The nozzle inserted in this extruder."
 msgstr "L’ugello inserito in questo estrusore."
 
 msgctxt "@label"
-msgid ""
-"The pattern of the infill material of the print:\n"
-"\n"
-"For quick prints of non functional model choose line, zig zag or lightning infill.\n"
-"\n"
-"For functional part not subjected to a lot of stress we recommend grid or triangle or tri hexagon.\n"
-"\n"
-"For functional 3D prints which require high strength in multiple directions use cubic, cubic subdivision, quarter cubic, octet, and gyroid."
-msgstr ""
-"La configurazione del materiale di riempimento della stampa:\n"
-"\n"
-"Per stampe rapide di modelli non funzionali scegli linea, zig zag o riempimento fulmine.\n"
-"\n"
-"Per le parti funzionali non soggette a forti sollecitazioni, si consiglia griglia o triangolo o tri-esagonale.\n"
-"\n"
+msgid "The pattern of the infill material of the print:\n\nFor quick prints of non functional model choose line, zig zag or lightning infill.\n\nFor functional part not subjected to a lot of stress we recommend grid or triangle or tri hexagon.\n\nFor functional 3D prints which require high strength in multiple directions use cubic, cubic subdivision, quarter cubic, octet, and gyroid."
+msgstr "La configurazione del materiale di riempimento della stampa:"
```

Comment:
Pattern is not "configurazione" but I can't find a proper translation litterally it should be "modello", maybe "schema"?

Reply:
Well, I can't tell you :smiley: the term "pattern" is also used for "Infill Pattern" and translated as "Configurazione", which is quite relevant in this case, but not very precise. If you think an other word suits better, we can change them both.

Reply:
Well somewhere in the WEB I found "Modello di riempimento" to translate "Infill pattern" then I think that could be the best translation I can suggest